CMakeLists.txt

```
@@ -67,6 +67,51 @@ find_package(Dpctl REQUIRED)
 message(STATUS "Dpctl_INCLUDE_DIR=" ${Dpctl_INCLUDE_DIR})
 message(STATUS "Dpctl_TENSOR_INCLUDE_DIR=" ${Dpctl_TENSOR_INCLUDE_DIR})
 
+option(DPNP_TARGET_CUDA
+    "Build DPNP to target CUDA devices"
+    OFF
+)
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION OFF)
+set(_dpnp_sycl_targets)
+set(_dpnp_use_onemkl OFF)
```

I\d propose to rename:
```suggestion
set(_use_onemkl_interface OFF)
```